From 58147959d7d9357ee2009084961fa1606ce38bc4 Mon Sep 17 00:00:00 2001
From: Thor Thayer <tthayer@opensource.altera.com>
Date: Fri, 22 Apr 2016 10:33:39 -0500
Subject: [PATCH 029/151] FogBugz #411698-4: socfpga: dts: Add A10-SR Devkit
 power supply alarms

commit  d9e01ca4392760f55fbb0a38bb6d3c35c9ce8592 from
https://github.com/altera-opensource/linux-socfpga.git
branch is socfpga-5.4.64-lts

Add the Altera Arria10 Devkit power supply alarms.
[backport 'commit c5c3b0edeffe ("FogBugz #411698-4: socfpga: dts:
                         Add A10-SR Devkit power supply alarms")']

Signed-off-by: Thor Thayer <tthayer@opensource.altera.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
Integrated-by: Jun Zhang <jun.zhang@windriver.com>
---
 arch/arm/boot/dts/socfpga_arria10_socdk.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/socfpga_arria10_socdk.dtsi b/arch/arm/boot/dts/socfpga_arria10_socdk.dtsi
index 2632c2e620fe..b3d99e434223 100644
--- a/arch/arm/boot/dts/socfpga_arria10_socdk.dtsi
+++ b/arch/arm/boot/dts/socfpga_arria10_socdk.dtsi
@@ -114,6 +114,10 @@
 			compatible = "altr,a10sr-reset";
 			#reset-cells = <1>;
 		};
+
+		ps_alarm {
+			compatible = "altr,a10sr-hwmon";
+		};
 	};
 };
 
-- 
2.26.1

